hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/AbiConstants.java

```diff
@@ -93,6 +93,12 @@ private AbiConstants() {
     public static final int ABI_ID_ERC_OWNER_OF_NFT = 0x6352211e;
     // tokenURI(uint256 tokenId)
     public static final int ABI_ID_ERC_TOKEN_URI_NFT = 0xc87b56dd;
+    // isFrozen(address, address)
+    public static final int ABI_IS_TOKEN_FROZEN = 0x46de0fb1;
+    // freezeToken(address, address)
+    public static final int ABI_FREEZE_TOKEN = 0x5b8f8584;
+    // unfreezeToken(address, address)
+    public static final int ABI_UNFREEZE_TOKEN = 0x52f91387;
```

Comment:
In order to be compliant with the other signature comments you can add variable names, not only types.

Basically the signature for `isFrozen` is `isFrozen(address,address)` but to have more context about the function and parameters we also have added parameter names like `from`, `to`, `tokenId`, `value` to have better understanding about the function. These parameter names, however, are not part of the signature itself and are not used to calculate the function selector.